fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/SkolemNodeRdfContext.java

```diff
@@ -93,6 +94,10 @@ public SkolemNodeRdfContext(final FedoraResource resource,
             return null;
 
         } catch (final RepositoryException e) {
+            if (e instanceof ItemNotFoundException) {
+                // Effectively skip resources that do not exist
```

Comment:
IF THIS PR MAKES IT IMPOSSIBLE TO CREATE SUCH LINKS, WHY DO YOU HAVE CODE ABOVE COMMENTED "skip resources that do not exist".